gcloud-java-pubsub/src/test/java/com/google/cloud/pubsub/PubSubImplTest.java

```diff
@@ -1191,149 +1229,209 @@ public void testListTopicSubscriptionsAsyncWithOptions()
   }
 
   @Test
+  public void testPullMessages() {
+    pubsub = new PubSubImpl(options, renewerMock);
+    PullRequest request = PullRequest.newBuilder()
+        .setSubscription(SUBSCRIPTION_NAME_PB)
+        .setMaxMessages(42)
```

Comment:
I'm presuming there is no significance  to this other than the obvious reference to HG2G.